templates/xcode.pkr.hcl

```diff
@@ -43,6 +43,26 @@ source "tart-cli" "tart" {
   ssh_timeout  = "120s"
 }
 
+locals {
```

Comment:
I'm not sure what's the point of introducing `locals` and `dynamic`, if we can continue using the `for version in var.xcode_version` as it was before? 🤔

It looks like we only need to change the `inline` instructions accordingly.

Reply:
I didn't find a way to have multiple lines there. Trying to squeeze it in one line is not maintainable IMO.